AgContext.yaml

```diff
@@ -0,0 +1,149 @@
+
+agcontext_name:
+  type:
+    anyOf:
```

Comment:
you don't need anyOf when there's a single option. `type: string` would have been sufficient.